modules/update_data_table.nf

```diff
@@ -20,12 +22,12 @@ process update_data_table {
    if (production) {
       """
       cd /root/scripts/
-      python update_data_table.py -p /root/scripts/config.json -c $cohort -m "$comment" primary -pd
+      python update_data_table.py -p /root/scripts/config.json -c $cohort -m "$comment" primary -pd -rp $replace_patient_tier1a -rs $replace_sample_tier1a
```

Comment:
Does this work if -rp and -rs are store_true type of arguments in update_data_table.py? I remember from my experience it won't accept an argument, and so I had to do a lot of[ if/else logic here](https://github.com/Sage-Bionetworks/GENIE-Sponsored-Projects/blob/develop/main.nf#L21-L50) to use the --use-grs flag which is store_true. Because propagating true / false values from nextflow -> python doesn't propagate correctly